File: 520-detect-capital/detect-capital.js
/**
 * @param {string} word
 * @return {boolean}
 */
var detectCapitalUse = function(word) {
    if (word == word.toUpperCase()){
        return true
    }else if(word == word.toLowerCase()){
        return true
    }else if(word == word.slice(0,1).toUpperCase()+word.slice(1).toLowerCase()){
        return true
    }
    return false
};

File: 520-detect-capital/detect-capital.py
class Solution:
    def detectCapitalUse(self, word: str) -> bool:
        if len(word) == 1:
            return True
        if word[0].isupper():
            if word[1].isupper():
                for i in word[2:]:
                    if i.islower():
                        return False
            else:
                for i in word[2:]:
                    if i.isupper():
                        return False
        else:
            for i in word[1:]:
                if i.isupper():
                    return False
        return True
